bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/gmos/GmosOiwfsProbeArm.scala

```diff
@@ -44,28 +45,25 @@ object GmosOiwfsProbeArm extends ProbeArmGeometry {
     new Rectangle2D.Double(xy, xy, PickoffMirrorSize, PickoffMirrorSize)
   }
 
-  override def armAdjustment(ctx0: ObsContext, guideStarCoords: Coordinates, offset0: Offset): Option[ArmAdjustment] = {
+  override def armAdjustment(ctx0: ObsContext,
+                             guideStarCoords: Coordinates,
+                             offset0: Offset,
+                             T: Point2D): Option[ArmAdjustment] = {
     import ProbeArmGeometry._
 
     for {
-      ctx       <- Option(ctx0)
-      offset    <- Option(offset0)
+      ctx <- Option(ctx0)
+      offset <- Option(offset0)
       wfsOffset <- ctx.getInstrument match {
         case gmosn: InstGmosNorth => Some(gmosn.getFPUnit.getWFSOffset)
```

Comment:
It is and is specified in `InstGmosCommon`, but has a parameterized return type of `P extends Enum[P] with GmosCommonType.FPUnit`, which is either a `GmosNorthType.FPUnit` or `GmosSouthType.FPUnit`. If I remember correctly, trying to combine these was giving me endless headaches and bizarre exceptions about parameterization and methods not being found. Such an annoyance!